### PR TITLE
TS fixes

### PR DESCRIPTION
**Pull request by matesich**
**Related issue(s):**
* Fixes TS issues

**Problem:**
* No tabs are selected when PDP open, first one should be selected by default
* Qty input on PDP does not look right
* Compare, wishlist is glued to qty output on pdp on narrow screens, must wrap with padding, or not wrap at all.
* On PDP, on narrow screens, while loading the gallery takes less width then when loaded, makes product details jump. Make gallery and loading gallery have same widht.
* MyAccount Tabs not working correctly
* When searching search item title is not vertically centered
* When comparing products text in description has no padding. Suggestion, add padding to each item, remove custom ones
* Product compare, the width of compare items is not 100% of the screen and seems to be hard to style. I would redo it, right not it only relies on browsers understanding of min-content? Right?
* Why is there 2 dividers in menu? Looks strange!
* When opening the popup, close action is dispatched immediately ( on desktop version )

**In this PR:**
* fixed issues listed above

### CODE REVIEW

File: packages/scandipwa/src/component/Notification/Notification.component.tsx [hunk 1]
```diff
@@ -107,11 +107,13 @@ export class Notification extends PureComponent<NotificationComponentProps, Noti
...
         };
 
+        const message = msgText.toString();
+
         return (
             <div block="Notification" mods={ mods } ref={ this.notification } id={ id }>
                 <button block="Notification" elem="Button" onClick={ this.hideNotification }> Close </button>
                 <p block="Notification" elem="Text">
-                    <Html content={ msgText } />
+                    <Html content={ message } />
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Notification/Notification.component.tsx:116]: Better to take a look into core of problem

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/Notification/Notification.component.tsx (hunk 1 above)
Comment (matesich) [reply packages/scandipwa/src/component/Notification/Notification.component.tsx]: resolved by this [PR](https://github.com/scandipwa/scandipwa/pull/5209)

**Review by alfredsgenkins — APPROVED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

File: packages/scandipwa/src/component/ProductTabs/ProductTabs.component.tsx [hunk 2]
```diff
@@ -23,34 +23,41 @@ import './ProductTabs.style';
...
-            const { tabs: [{ id }] } = this.props;
-
-            this.state = {
-                activeTab: id,
-            };
-        } else {
-            this.state = {
-                activeTab: '',
-            };
-        }
```
Comment (alfredsgenkins) [inline packages/scandipwa/src/component/ProductTabs/ProductTabs.component.tsx:42]: Are these changes moved to `this.updateDefaultSelectedTab`, if yes all good.

**Review by alfredsgenkins — APPROVED** [review]
Looks good to mw

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**